lib/pure/collections/tables.nim

```diff
@@ -131,7 +131,7 @@
 ##   let myString = "abracadabra"
 ##   let letterFrequencies = toCountTable(myString)
 ##   echo letterFrequencies
-##   # 'a': 5, 'b': 2, 'c': 1, 'd': 1, 'r': 2}
+##   # output: { 'a': 5, 'b': 2, 'c': 1, 'd': 1, 'r': 2}
```

Comment:
One extra space， should be `{'a': 5, 'b': 2, 'c': 1, 'd': 1, 'r': 2}`

Reply:
going to force push...